Models/Movie.cs

```diff
@@ -2,17 +2,32 @@
 
 namespace afh_be.Models
 {
-    public class Movie
+    /* public class Movie
     {
         [DatabaseGenerated(DatabaseGeneratedOption.None)]
-        public int MovieID { get; set; }
+        [ForeignKey ("MovieID")]
```

Comment:
I was intending to keep it for reference, in case we want to revert anything I've done. But I guess we can just look at a previous commit for the same purpose!